CHANGELOG.md

```diff
@@ -38,6 +71,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Fix bug that users who are connected to another extension would hit when viewing connected sites ([#13974](https://github.com/MetaMask/metamask-extension/pull/13974))
 
+
```

Comment:
This looks like a mistake. I would expect this to cause the changelog validation to fail.

Reply:
This still seems like a validation bug to me but this doesn't need to block of course.